users/users-router.js

```diff
@@ -23,8 +23,8 @@ router.post('/:id/tasks', restricted, (req, res) => {
   console.log(id)
```

Comment:
Title should read "Adding task will now return the task id"

Reply:
Nice fix -- totally needed.